scripts/packages/bazel.sh

```diff
@@ -191,7 +191,7 @@ if [[ ! -x $BAZEL_REAL ]]; then
 fi
 
 readonly wrapper="${workspace_dir}/tools/bazel"
-if [[ -x "$wrapper" ]]; then
+if [[ -x "$wrapper" && ! -d "$wrapper" ]]; then
```

Comment:
Code review brought up the question of why not just use `-f` here instead of `! -d`:

```
if [[ -x "$wrapper" && -f "$wrapper" ]]; then
```

It looks fine to me, because symlinks are handled correctly that way, too ("If file is a symbolic link, test will fully dereference it and then evaluate the expression against the file referenced"), but wanted to check with you.

Reply:
Honestly I just went with what it was before, so I didn’t really think about alternatives. If there’s a preferred change, I’m fine with that 👍

Reply:
I’m also not able to make the changes to the pr currently, so feel free to edit it yourself if you want